lib/Dialect/CGGI/Transforms/Passes.td

```diff
@@ -19,4 +19,20 @@ def SetDefaultParameters : Pass<"cggi-set-default-parameters"> {
   let dependentDialects = ["mlir::heir::cggi::CGGIDialect"];
 }
 
+def BooleanLineVectorizer : Pass<"cggi-boolean-line-vectorize"> {
+  let summary = "Group different logic gates with the packed API";
+  let description = [{
+    This pass groups indepentend logic gates into a single call of the packed
+    operations.
+
+    Pass is intended for the `FPT` tfhe-rs API.
+  }];
+
+  let dependentDialects = [
+    "tensor::TensorDialect",
+  ];
+
+  let dependentDialects = ["mlir::heir::cggi::CGGIDialect"];
```

Comment:
I think you can get away without cggi as a dependent dialect since it must have already been loaded as a dialect to parse the input. (https://mlir.llvm.org/getting_started/Faq/#registered-loaded-dependent-whats-up-with-dialects-management)